cyclonedx_buildroot/_internal/cli.py

```diff
@@ -174,22 +174,21 @@ def run(*, argv: Optional[Sequence[str]] = None, **kwargs: Any) -> Union[int, No
     br_bom = Bom()
     br_bom.metadata.component = rootComponent = Component(name=args.product_name,
                                                           version=args.product_version)
-    # TODO Specification 1.6 ends support of "manufacture". Remove in future release.
-    br_meta = BomMetaData(manufacture=OrganizationalEntity(name=args.manufacturer_name),
-                          manufacturer=OrganizationalEntity(name=args.manufacturer_name),
+
+    br_meta = BomMetaData(manufacturer=OrganizationalEntity(name=args.manufacturer_name),
```

Comment:
what is this args default value of `'undefined'` strings nonsense near line 160? 
> `parser.add_argument('-m', action='store', dest='manufacturer_name', default='unknown',
help='name of product manufacturer')`

python knows`None` - a special type, similar to `NULL` - better use it to indicate that a user did not give n answer. ...

and here: 

```suggestion
   if args.manufacturer_name: 
     br_bom.metadata.manufacturer=OrganizationalEntity(name=args.manufacturer_name)
```

Reply:
The default values of "unknown" are searchable in the output file. This is a feature I have used to determine if the output has bad default values. I use it as a quality metric. Setting to a None value will require me to review the potential issues for such a change. I can create a new issue to track a review of the feature request you propose.

Reply:
https://github.com/CycloneDX/cyclonedx-buildroot/issues/75

Reply:
all thse switches are optional, right, and they default to 'unknown'.
how wouod it make sense to have an optional value with an invalid/undesireable behaviour, just to manuallytestthe results for these values?

Reply:
as long as you are stuck with this 'unknown',
you should go with something like 
```suggestion
   if args.manufacturer_name !== 'unknown': 
     br_bom.metadata.manufacturer=OrganizationalEntity(name=args.manufacturer_name)
```

Reply:
Issue 75 can handle this proposal. From a customer perspective and a maintenance perspective I will leave this in place.

Reply:
I suggest we leave the code unchanged. My long history in the embedded space made it clear that every addition of conditional code (if, then, else, etc.) adds complexity. Testing every code branch to prove nothing is broken is a serious need. So I suggest we leave this style for now. Feel free to submit a separate feature with code changes and the resulting numerous for each branch.